CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ This file itself is based on [Keep a CHANGELOG](https://keepachangelog.com/en/0.
 
 ### Changed
 - Upgrade for Kubernetes 1.17 (#115).
+- topolvm-controller requires `--leader-election-id` flag.
```

Comment:
No longer required as it has a default value.

Reply:
TopoLVM v0.4.1 requires `--leader-election-id` and v0.4.2(next) set default value on it.

Reply:
I see.

Reply:
@ymmt2005 
Do you mean that we do not need to tweak the changelog v0.4.1?

Reply:
?  This was resolved already.

Reply:
We are going to include this fix in v0.4.2.

Reply:
@ymmt2005 

> ? This was resolved already.

Do you mean that this PR is LGTM?

Reply:
Not actually took a look.  But LGTM.